EveripediaNetwork/ep-ui
# Fixed sidebar positioning

# UI Accessibility and Navigation Issues

_This pr fixes sidebar positioning and account menu scrolling issue_

## How should this be tested?

1. Verify the actions sidebar on wiki details page is visible
2. Click on the avatar icon from the main header, and verify the ui is scrollable

## Linked issues

closes https://github.com/EveripediaNetwork/issues/issues/3020
